content/doc/pipeline/index.adoc

```diff
@@ -1,981 +1,437 @@
+
+== Approaches to Defining Pipeline Script
+You can create pipelines in either of the following ways:
+
+* Through script entered in the configuration page of the user interface for your Jenkins instance.
+* Through script that you create with a Groovy editor outside Jenkins but import into a designated Jenkins repository by selecting the *Pipeline Script from SCM* option during initial setup.
+
+== Creating a Simple Pipeline
+
+Initial pipeline usage typically involves the following tasks:
+
+. Downloading and installing the Pipeline plugin (Unless it is already part of your Jenkins  installation)
+. Creating a Pipeline of a specific type
+. Configuring your Pipeline
+. Controlling Flow through your Pipeline
+. Scaling your Pipeline
+
+To create a simple pipeline from the Jenkins interface, perform the following steps:
+
+. Click *New Item* on your Jenkins home page,  enter a name for your (pipeline) job, select *Pipeline*, and click *OK*.
+. In the Script text area of the configuration screen, enter your pipeline syntax. If you are new to pipeline creation, you might want to start by opening Snippet Generator and selecting the “Hello Word” snippet.
+*Note:* Pipelines are written as Groovy scripts that tell Jenkins what to do when they are run, but because relevant bits of syntax are introduced as needed, you do not need deep expertise in Groovy to create them, although basic understanding of Groovy is helpful.
+. Check the Use Groovy Sandbox option below the Script text area.
+*Note:* If you are a Jenkins administrator (in other words, authorized to approve your own scripts), sandboxing is optional but efficient, because it lets scripts run without approval as long as they limit themselves to operations that Jenkins considers inherently safe.
+. Click *Save*.
+. Click *Build Now* to create the pipeline. 
+. Click ▾ and select *Console Output* to see the output. 
 
-### Scripting
+The following example shows a successful build of a pipeline created with a one-line script that uses the “echo” step to output the phrase, “hello from pipeline:”
 
-Jenkins Pipeline supports executing shell (*nix) or batch scripts (Windows) just like freestyle jobs:
+  Started by user anonymous
+  [Pipeline] echo
+  hello from Pipeline
+  [Pipeline] End of Pipeline
+  Finished: SUCCESS
 
-```groovy
-sh 'sleep 10'
-```
+*Note:* You can also create complex and multi-branch pipelines in the script entry area of the Jenkins configuration page, but because they contain multiple stages and the configuration page UI provides limited scripting space, pipeline creation is more commonly done using an editor of your choice from which scripts can be loaded into Jenkins using the *Pipeline script from SCM* option.
 
-```groovy
-bat 'timeout /t 10'
-```
+== Creating Multi-branch Pipelines
```

Comment:
This section should mention that the "PIpeline Mulltibranch" plugin is required for this section. Perhaps an `INFO:` block?